components/atom/tag/src/index.scss

```diff
@@ -1,11 +1,11 @@
 @import '~@schibstedspain/sui-theme/lib/settings';
 @import '~@schibstedspain/sui-theme/lib/index';
 
-$bgc-AtomTag: color-variation($c-gray, 3) !default;
-$c-AtomTagActionable: $c-white !default;
-$h-AtomTag: $p-xxl !default;
+$bgc-atom-tag: color-variation($c-gray, 3) !default;
+$c-actionable-atom-tag: $c-white !default;
```

Comment:
according to convention it'll be `$c-atom-tag-actionable`

Reply:
fixed! 👍